modules/menu/css/lib/menu.scss

```diff
@@ -40,32 +43,72 @@
       cursor: pointer;
       transition: background-color 80ms;
 
-      a {
-        color: inherit;
-        text-decoration: none;
-      }
-
-      &:focus,
       &:hover {
         background-color: $wdc-color-common-hover-background;
+
+        .wd-icon-fill {
+          fill: $wdc-color-icon-hover;
+        }
       }
 
       &.wdc-menu-item-disabled {
         color: $wdc-color-licorice-100;
         cursor: default;
 
+        .wd-icon-fill {
+          fill: $wdc-color-icon-disabled;
+        }
+
         &:hover {
           background-color: transparent;
         }
+      }
+
+      &.wdc-menu-item-focused {
+        background-color: $wdc-color-common-focus-background;
 
-        &.wdc-menu-item-focused {
+        &.wdc-menu-item-disabled {
           background-color: $wdc-color-blueberry-200;
         }
       }
 
-      &.wdc-menu-item-focused {
-        background-color: $wdc-color-common-focus-background;
+      &.wdc-menu-item-focused,
+      &.wdc-menu-item-focused.wdc-menu-item-disabled {
         color: $wdc-color-type-inverse;
+
+        .wd-icon-fill {
+          fill: $wdc-color-icon-inverse;
+        }
+      }
+
+      a {
+        color: inherit;
+        text-decoration: none;
+      }
+
+      .wd-icon-fill {
+        fill: $wdc-color-icon-standard;
+      }
```

Comment:
Is there a code style guideline I should follow when it comes to nesting? Right now, I have:

```
.wdc-menu {
  ul {
    li {
      .wd-icon-fill {
        ...
      }

      .wdc-icon,
      .wdc-menu-item-icon {
         ...
      }
    }
  }
}
```

I could reduce the amount of nesting and do this instead:

```
.wdc-menu {
  ul {
    li {
      ...
    }
  }

  .wd-icon-fill {
    ...
  }

  .wdc-icon,
  .wdc-menu-item-icon {
     ...
  }
}
```

In general, I prefer less nesting (even though that's not what I've currently done, whoops) since I find it easier to read/debug, but I'm happy to follow whatever guidelines we've set for the project.

Reply:
Got it, I'll leave this as is then.